presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java

```diff
@@ -276,9 +278,9 @@ void nextAttempt()
         private final CounterStat success = new CounterStat();
         private final CounterStat finalFailure = new CounterStat();
         private final DistributionStat attemptsBeforeFailure = new DistributionStat();
-        private final DistributionStat failureLatencyMillis = new DistributionStat();
+        private final TimeStat failureLatency = new TimeStat(MICROSECONDS);
```

Comment:
Why do you choose to use microseconds as a unit instead of, say, milliseconds? The original scale of values is nanoseconds. More typical way of expressing latency is in milliseconds.